### DISCUSSION TITLE
Mac Big Sur always get compile error.

### DISCUSSION THREAD

**Discussion by Doris1039** [microsoft/AirSim#3335] [Support Q&A]
After I updated my system to Big Sur, I find out I need to use Unreal Engine 4.25 instead of the 4.24 according to the AirSim website. But when I open the Blocks file, it always compiles error. 

I noticed that it says AirSim only supports Only macOS Catalina (10.15), so I cannot use it in Big Sur?

**Comment by thiago-cavalcante** [comment]
Did you find out a workaround to run on macOS > Catalina (10.15)?
